etp-backend/src/main/sql/solita/etp/db/valvonta-oikeellisuus.sql

```diff
@@ -219,3 +219,6 @@ delete from vo_tiedoksi where toimenpide_id = :toimenpide-id;
 
 -- name: delete-draft-toimenpide!
 update vo_toimenpide set deleted = true where id = :toimenpide-id and publish_time is null;
+
+-- name: update-default-valvoja!
+update energiatodistus set valvonta$valvoja_id = :whoami-id where id = :id and valvonta$valvoja_id is null;
```

Comment:
If your text editor supports always terminating every line with a trailing `\n`, would be nice to add that.